tests/console/fetchmail_ssl.pm

```diff
@@ -0,0 +1,64 @@
+# SUSE's openQA tests
+#
+# Copyright © 2017 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+# Summary: Test fetchmail works with SSL enabled
+# Note: fetchmail connects to a remote mail server (running dovecot)
+#   and fetch mails to localhost, then deliver mails (by postfix) to
+#   local mailbox.
+# Maintainer: Qingming Su <qmsu@suse.com>
+
+use base "consoletest";
+use strict;
+use testapi;
+use utils;
+use mailtest;
+
+sub run {
+    my $self        = shift;
+    my $fetchmailrc = "~/.fetchmailrc";
+    my $test_cacert = "~/ca-cert.pem";
+    my $mailbox     = "/var/mail/$username";
+    my $subject     = "Testing fetchmail";
+
+    select_console "root-console";
+    prepare_mail_client;
+
+    # Install postfix and fetchmail
+    zypper_call "in postfix fetchmail";
+    systemctl "start postfix.service";
+    postfix_dns_lookup_off if $mail_server_ip;
+    postfix_config_update;
+    assert_script_run "curl " . data_url('openssl/ca-cert.pem') . " -o $test_cacert";
+
+    # Need mailx to send testing mail
+    mailx_setup(ssl => "yes", host => $mail_server_name);
+
+    # Switch to user console
+    type_string "chown $username /dev/$serialdev\n";
+    select_console "user-console";
+
+    for my $protocol (qw/pop3 imap/) {
```

Comment:
try `qw(pop3 imap)`

Reply:
Thanks, hope this solves the CI check error.